### ISSUE TITLE
TextEditor.onDidChangeScrollTop is deprecated.

### ISSUE THREAD

**Issue by WickedSik** [mehcode/atom-minimap-plus#5]
This is now a view method. Call TextEditorElement::onDidChangeScrollTop instead.
```
TextEditor.onDidChangeScrollTop (<embedded>:2945:18)
<unknown> (/Users/jurriendokter/.atom/packages/minimap-autohide/lib/minimap-autohide.coffee:32:49)
editorsMinimaps.forEach (/Users/jurriendokter/.atom/packages/minimap-plus/lib/main.js:377:51)
Map.forEach (native)
Main.observeMinimaps (/Users/jurriendokter/.atom/packages/minimap-plus/lib/main.js:377:28)
Object.activatePlugin (/Users/jurriendokter/.atom/packages/minimap-autohide/lib/minimap-autohide.coffee:27:55)
```

**Comment by mehcode** [comment]
@WickedSik It seems https://github.com/jayk/minimap-autohide was deprecated in favor of https://atom.io/packages/minimap-autohider

The later works great with this package with no deprecations.

**Comment by WickedSik** [comment]
I'll move on to that package then :)

**Comment by mehcode** [comment]
Note that there _may_ have been a problem before I fixed #2 (so be sure to update as the fix is out) as I looked at that issues first.

**Comment by WickedSik** [comment]
I'll be sure to update indeed, and I may check it again when updates have arrived. (Unless I forget, feel free to remind me)
